### ISSUE TITLE
Soundcloud using 64 kbps Opus codec

### ISSUE THREAD

**Issue by winterflags** [flyingrub/scdl#251]
I recently saw this article: https://www.magneticmag.com/2018/01/soundcloud-lowers-sound-quality-uploads-half/

It claims that Soundcloud has lowered the sound quality, from using previous 128 kbps MP3 to 64 kbps Opus codec.

However, the article was later updated with a statement from Soundcloud:

> A SoundCloud after many emails back and forth refutes the claim that the sound quality has not changed and the reports are "inaccurate." "We haven't altered our approach to audio quality," continues a statement from SC. "We have been using the Opus codec (among others) since 2016, and we regularly test different combinations of encoding and streaming to offer listeners a quality experience on any device."

My question is not an scdl issue per se, but I'm curious if the scdl community has any insight into the subject and what the actual quality options for 'ripping' are.

A Google Chrome plugin author claims that his plugin forces the browser to play 128 kbps MP3 instead of Opus: https://chrome.google.com/webstore/detail/soundcloud-quality-restor/llhggdgjolganlijhnocpbjfmdomanlc?hl=en

**Comment by flyingrub** [comment]
Just tested some random track and didn't get any 64Kbps Opus one. I think someone would have reported it here if he got some 64kbps one with scdl ?

**Comment by justinTM** [comment]
@flyingrub there is no chance the file meta data will say '256kbps' but **actually be** 64, right?

**Comment by flyingrub** [comment]
Nope, this would be totally useless has we would have a bigger file with a lower sound quality.
